ArduPlane/quadplane.cpp

```diff
@@ -2562,7 +2562,11 @@ bool QuadPlane::is_vtol_takeoff(uint16_t id) const
 bool QuadPlane::is_vtol_land(uint16_t id) const
 {
     if (id == MAV_CMD_NAV_VTOL_LAND) {
-        return true;
+        if (options & QuadPlane::OPTION_MISSION_LAND_FW_APPROACH) {
+          return plane.vtol_approach_s.approach_stage == Plane::Landing_ApproachStage::VTOL_LANDING;
```

Comment:
indent is odd

Reply:
My bad, I use 2 spaces normally on all my own stuff, and missed that it was using the wrong value here.